src/components/GithubLink/GithubLink.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../sass/utils/variables';
+
+.github {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+
+  &__link {
+    transition: all 300ms;
+    &:hover {
+      filter: opacity(80%);
```

Comment:
I think a good practice with transitions is to set both the old state and the new state. So in this case, add `filter: opacity(100%)` to the `github__link`